isthmus/src/main/java/io/substrait/isthmus/IsthmusEntryPoint.java

```diff
@@ -17,14 +20,18 @@
 @Command(
     name = "isthmus",
     version = "isthmus 0.1",
-    description = "Converts a SQL query to a Substrait Plan")
-public class PlanEntryPoint implements Callable<Integer> {
+    description = "Substrait Java Native Image for parsing SQL Query and SQL Expressions")
+public class IsthmusEntryPoint implements Callable<Integer> {
+  static final org.slf4j.Logger logger = org.slf4j.LoggerFactory.getLogger(IsthmusEntryPoint.class);
 
-  static final org.slf4j.Logger logger = org.slf4j.LoggerFactory.getLogger(PlanEntryPoint.class);
-
-  @Parameters(index = "0", description = "The sql we should parse.")
+  @Parameters(index = "0", arity = "0..1", description = "The sql we should parse.")
   private String sql;
 
+  @Option(
+      names = {"-e", "--sqlExpression"},
```

Comment:
Can we name the option `--expression` or `--extendedexpression` instead?

Reply:
Changed.